gnmi_server/gnoi_system_test.go

```diff
@@ -0,0 +1,261 @@
+package gnmi
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/sonic-net/sonic-gnmi/common_utils"
+
+	syspb "github.com/openconfig/gnoi/system"
+	"github.com/openconfig/gnoi/types"
+	"github.com/redis/go-redis/v9"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/status"
+
+)
+
+func errorCodeToSwss(errCode codes.Code) string {
+	switch errCode {
+	case codes.OK:
+		return "SWSS_RC_SUCCESS"
```

Comment:
>SWSS_ [](http://example.com/codeflow?start=10&length=5)

I am wondering why choosing SWSS_RC_ as prefix. SWSS stands for SWitch State Service. Here is more general linux reboot or general operations.

Reply:
This is coming from the existing SONiC implementation where this format for error codes is recognized by the reboot backend (since it uses swsscommon libraries) and this translation is between gRPC codes and SONiC definitions